packages/wallets/src/components/Base/WalletButtonGroup/WalletButtonGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { type WalletButton } from '../WalletButton';
+import './WalletButtonGroup.scss';
+import classNames from 'classnames';
+
+type TWalletButtonGroupProps = {
+    isVertical?: boolean;
+};
+
+const WalletButtonGroup: React.FC<React.PropsWithChildren<TWalletButtonGroupProps>> = ({ children, isVertical }) => {
+    return (
+        <div
+            className={classNames('wallets-button-group', {
+                'wallets-button-group--vertical': isVertical,
+            })}
+        >
+            {Children.map(children, child => (
+                <div className='wallets-button-group__item'>{child}</div>
+            ))}
+        </div>
+    );
+};
+
+export default WalletButtonGroup;
```

Comment:
Added a new base component to group buttons.
<img width="375" alt="Screenshot 2023-10-23 at 1 30 51 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/fc150f43-dc2e-4c35-8c27-ba37edb55a10">

<img width="490" alt="Screenshot 2023-10-23 at 1 30 57 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/329f722a-69e1-4864-9346-9484e23c7e54">

Adding `isVertical` option allows the button grouping to be grouped vertically:
<img width="485" alt="Screenshot 2023-10-23 at 1 31 04 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/8499142b-e5db-420a-88d4-f894c3588a4b">